tests/EnumSchemaValidatorTest.php

```diff
@@ -17,16 +17,16 @@ final class EnumSchemaValidatorTest extends BaseCodegenTestCase {
   public function testStringEnum(): void {
```

Comment:
This file was completely unclear if it cares about array types, defaulting to darrays.